api/types/types.go

```diff
@@ -63,3 +64,91 @@ type Code struct {
 	Language string   `bson:"language" json:"language"`
 	Files    []string `bson:"files" json:"files"`
 }
+
+// HuskyCIVulnerability is the struct that stores vulnerability information.
+type HuskyCIVulnerability struct {
+	Language       string `bson:"language" json:"language,omitempty"`
+	SecurityTool   string `bson:"securitytool" json:"securitytool,omitempty"`
+	Severity       string `bson:"severity,omitempty" json:"severity,omitempty"`
+	Confidence     string `bson:"confidence,omitempty" json:"confidence,omitempty"`
+	File           string `bson:"file,omitempty" json:"file,omitempty"`
+	Line           string `bson:"line,omitempty" json:"line,omitempty"`
+	Code           string `bson:"code,omitempty" json:"code,omitempty"`
+	Details        string `bson:"details" json:"details,omitempty"`
+	Type           string `bson:"type,omitempty" json:"type,omitempty"`
+	VunerableBelow string `bson:"vulnerablebelow,omitempty" json:"vulnerablebelow,omitempty"`
+	Version        string `bson:"version,omitempty" json:"version,omitempty"`
+	Occurrences    int    `bson:"occurrences,omitempty" json:"occurrences,omitempty"`
+}
+
+// HuskyCIResults is a struct that represents huskyCI scan results.
+type HuskyCIResults struct {
+	GoResults         GoResults         `bson:"goresults,omitempty" json:"goresults,omitempty"`
+	PythonResults     PythonResults     `bson:"pythonresults,omitempty" json:"pythonresults,omitempty"`
+	JavaScriptResults JavaScriptResults `bson:"javascriptresults,omitempty" json:"javascriptresults,omitempty"`
+	RubyResults       RubyResults       `bson:"rubyresults,omitempty" json:"rubyresults,omitempty"`
+}
+
+// GoResults represents all Golang security tests results.
+type GoResults struct {
+	HuskyCIGosecOutput HuskyCIGosecOutput `bson:"gosecoutput,omitempty" json:"gosecoutput,omitempty"`
+}
+
+// PythonResults represents all Python security tests results.
+type PythonResults struct {
+	HuskyCIBanditOutput HuskyCIBanditOutput `bson:"banditoutput,omitempty" json:"banditoutput,omitempty"`
+	HuskyCISafetyOutput HuskyCISafetyOutput `bson:"safetyoutput,omitempty" json:"safetyoutput,omitempty"`
+}
+
+// JavaScriptResults represents all JavaScript security tests results.
+type JavaScriptResults struct {
+	HuskyCIRetireJSOutput HuskyCIRetireJSOutput `bson:"retirejsoutput,omitempty" json:"retirejsoutput,omitempty"`
+	HuskyCINpmAuditOutput HuskyCINpmAuditOutput `bson:"npmauditoutput,omitempty" json:"npmauditoutput,omitempty"`
+}
+
+// RubyResults represents all Ruby security tests results.
+type RubyResults struct {
+	HuskyCIBrakemanOutput HuskyCIBrakemanOutput `bson:"brakemanoutput,omitempty" json:"brakemanoutput,omitempty"`
+}
+
+// HuskyCIGosecOutput stores all Low, Medium and High Gosec vulnerabilities
+type HuskyCIGosecOutput struct {
```

Comment:
Maybe we don't need all these structs since each of them store security tests results already in huskyCI format, and this format is the same for all security tests. We could have just one struct called `HuskyCIOutput`.

Reply:
Sounds great, @Takehime! Great point indeed. We could use `HuskyCISecurityTestOutput`, what do you think? 🙃

Reply:
Sounds good to me!

Reply:
I made this change here: ee94283

Reply:
Awesome! 🔥🎉